### PR TITLE
74 see user owned items in bag

### PR DESCRIPTION
**Pull request by Ferejbo**
Reliant on backend branch: item_ownership_endpoint_with_user_id

### CODE REVIEW

File: src/screens/BagScreen.tsx [hunk 1]
```diff
@@ -1,28 +1,76 @@
...
+import { useRecoilState } from "recoil";
+import getItemOwnerships from "../api/get-item-ownerships";
+import { ItemOwnership } from "../client";
+import { currentUser } from "../recoil/atom";
 
 const BagScreen = () => {
+  const [user] = useRecoilState(currentUser);
+  const { data, isLoading, isError } = useQuery<ItemOwnership[]>(
+    ["item-ownerships", user?.id],
+    () => getItemOwnerships(user ? user.id : "asd") //Very quick fix for potensial null value for user
```
[the remark this answers is not in the document]
Comment (Ferejbo) [reply src/screens/BagScreen.tsx:19]: Much better

File: src/screens/BagScreen.tsx [hunk 2]
```diff
@@ -1,28 +1,76 @@
...
       <Text style={styles.title}>{title}</Text>
-      <Text style={styles.subtitle}>{title}</Text>
+      <Text style={styles.subtitle}>
+        {description ? description : "This is an item"}
+      </Text>
     </View>
   );
 
+  if (isLoading) {
+    <ActivityIndicator animating={true} color={"purple"} />;
```
Comment (mathiazom) [inline src/screens/BagScreen.tsx:53]: Don't you need a `return` here?

Comment (Ferejbo) [reply src/screens/BagScreen.tsx]: Ooops, fixed both suggestions in latest commit

**Review by jonasoo98 — APPROVED** [review]

**Comment by jonasoo98** [comment]
Looks good to me, but maybe we should make it work with the main channel?

**Review by mathiazom — APPROVED** [review]

**Pull request merged**